### DISCUSSION TITLE
Continue if / stop if - how to use ??

### DISCUSSION THREAD

**Discussion by mtspl** [Budibase/budibase#7708] [Help]
HI. So im struggling with Continue if action. What i would like to achieve, is when one of form fields does not pass a validation test, condition is met and remaining part of defined actions would be processed. 

I have tried several different ways. Validate form and then, Continue if does not work properly, despite one of the field clearly indicates that custom validation conditions are violated. I also tried to pass the same custom validation java script into Continue statement, but it doest seem to be able to properly resolve values comparison. i tried returning boolean values, text and numeric values. None of those approaches lead me to success. 

I also update a field with the same result that the Continue if conditions are feed, and in each case, i received expected value, either true/false or other, expected text/numeric values. 

Any help would be appreciated.

**Comment by mtspl** [comment]
I have noticed one interesting detail. When data not violating validation rules are passed to the form, at firs, alert about field violation appears. Probably due to quite long response time (data is fetched via REST). After a short while, validation alert disappears, but the remaining part of Continue if, is already executed.

**Comment by melohagan** [comment]
Hey @mtspl 

Could you share some screenshots of what you have tried please? Also feel free to include an app export and I can take a look.
